jdaviz/configs/imviz/helper.py

```diff
@@ -26,6 +41,157 @@ def load_data(self, data, parser_reference=None, **kwargs):
 
         self.app.load_data(filepath, parser_reference=parser_reference, **kwargs)
 
+    # astrowidgets ducktyping (MVP)
+
+    def center_on(self, point):
+        raise NotImplementedError
+
+    def offset_to(self, dx, dy, skycoord_offset=False):
+        raise NotImplementedError
+
+    @property
+    def zoom_level(self):
+        raise NotImplementedError
+
+    @zoom_level.setter
+    def zoom_level(self, val):
+        raise NotImplementedError
+
+    def zoom(self, val):
+        raise NotImplementedError
+
+    @property
+    def stretch_options(self):
+        raise NotImplementedError
+
+    @property
+    def stretch(self):
+        raise NotImplementedError
+
+    @stretch.setter
+    def stretch(self, val):
+        raise NotImplementedError
+
+    @property
+    def autocut_options(self):
+        raise NotImplementedError
+
+    @property
+    def cuts(self):
+        raise NotImplementedError
+
+    @cuts.setter
+    def cuts(self, val):
+        raise NotImplementedError
+
+    @property
+    def colormap_options(self):
+        raise NotImplementedError
+
+    def set_colormap(self, cmap):
+        raise NotImplementedError
+
+    @property
+    def click_center(self):
+        return False  # No way to do this right now
+
+    @click_center.setter
+    def click_center(self, val):
+        raise NotImplementedError
+
+    @property
+    def click_drag(self):
+        return False  # No way to do this right now
+
+    @click_drag.setter
+    def click_drag(self, value):
+        raise NotImplementedError
+
+    @property
+    def scroll_pan(self):
```

Comment:
I first red this as `scroll_pun`, and have to say I'm a bit disappointed in the reality that it's `pan`

Reply:
We can have a `scroll_pun` mode where the viewer just spews puns in the "snackbar" as you scroll...

Reply:
Also removed from MVP, see https://github.com/spacetelescope/jdaviz/pull/632/files#r683793189